spec/factories/line_item.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+    factory :line_item do
+      dish_id { }
```

Comment:
```suggestion
      dish { create(:dish) }
```
Apply that for all relations in factories